### PR TITLE
Improve Autograd Documentation Clarity

### PR DESCRIPTION
**Pull request by ArekSredzki**
This makes minor adjustments to the autograd docs, improving clarity and resolving grammatical errors

cc @svekars @carljparker

### CODE REVIEW

File: docs/source/notes/autograd.rst [hunk 1]
```diff
@@ -164,16 +164,16 @@ of the module's parameters (which have ``requires_grad=True`` by default).
 Grad Modes
 ^^^^^^^^^^
 
-Apart from setting ``requires_grad`` there are also three possible modes
-enableable from Python that can affect how computations in PyTorch are
+Apart from setting ``requires_grad`` there are also three grad modes that can
+selected from Python that can affect how computations in PyTorch are
```
Comment (svekars) [inline docs/source/notes/autograd.rst:168]: did you mean "that can be selected"?

File: docs/source/notes/autograd.rst [hunk 2]
```diff
@@ -440,7 +440,7 @@ It also turns out that no interesting real-valued objective fulfill the
 Cauchy-Riemann equations. So the theory with homomorphic function cannot be
 used for optimization and most people therefore use the Wirtinger calculus.
 
-Wirtinger Calculus comes in picture ...
+Wirtinger Calculus comes into the picture ...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (svekars) [inline docs/source/notes/autograd.rst:444]: please make line 444 as long as line 443

File: docs/source/notes/autograd.rst (hunk 1 above)
Comment (ArekSredzki) [reply docs/source/notes/autograd.rst:168]: Good catch, yes :)

**Review by svekars — COMMENTED** [review]
LGTM

File: docs/source/notes/autograd.rst [hunk 3]
```diff
@@ -368,9 +368,9 @@ Thread Safety on Autograd Node
 
 Since Autograd allows the caller thread to drive its backward execution for
 potential parallelism, it's important that we ensure thread safety on CPU with
-parallel backwards that share part/whole of the GraphTask.
+parallel ``backward()`` calls that share part/whole of the GraphTask.
 
-Custom Python ``autograd.Function`` is automatically thread safe because of GIL.
+Custom Python ``autograd.Function``s are automatically thread safe because of GIL.
```
Comment (kit1980) [inline docs/source/notes/autograd.rst:373]: This change broke the rendering, compare https://github.com/pytorch/pytorch/blob/a9b17decd7af4fbe1295b112d9c2d255fb1e0fdd/docs/source/notes/autograd.rst#thread-safety-on-autograd-node and https://github.com/pytorch/pytorch/blob/master/docs/source/notes/autograd.rst#thread-safety-on-autograd-node

**Review by kit1980 — CHANGES_REQUESTED** [review]
Please make sure the document renders correctly after the changes.

File: docs/source/notes/autograd.rst (hunk 3 above)
Comment (ArekSredzki) [reply docs/source/notes/autograd.rst]: Good catch, sorry about that. It's now fixed

**Review by kit1980 — APPROVED** [review]

**Comment by ArekSredzki** [comment]
Thanks for trying to merge this @kit1980. It's not clear to me why some of the CI stages failed, does anything stand out to you?

**Comment by kit1980** [comment]
> Thanks for trying to merge this @kit1980. It's not clear to me why some of the CI stages failed, does anything stand out to you?

Probably some flakiness.  I've restarted the failed jobs.

**Pull request closed without merging**